src/train.py

```
@@ -28,9 +28,73 @@
+        return False
+
+    try:
+        # Set tracking URI
+        tracking_uri = os.environ.get("MLFLOW_TRACKING_URI")
+        mlflow.set_tracking_uri(tracking_uri)
+
+        # Set experiment
+        experiment_id = os.environ.get("MLFLOW_EXPERIMENT_ID")
+        mlflow.set_experiment(experiment_id=experiment_id)
+
+        print(f"MLflow configured successfully:")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)